source/_integrations/iotty.markdown

```diff
@@ -4,13 +4,15 @@ description: Instructions on how to connect iotty Smart Devices to Home Assistan
 ha_release: '2024.8'
 ha_category:
   - Switch
+  - Cover
 ha_iot_class: Cloud Polling
 ha_config_flow: true
 ha_codeowners:
   - '@pburgio'
 ha_domain: iotty
 ha_platforms:
   - switch
+  - cover
```

Comment:
I realize it is outside the strict scope of this PR, but could we move the Prerequisite section before the configuration? It makes more sense there.

Reply:
What do you mean exactly by "configuration"? Is this the config flow?

`{% include integrations/config_flow.md %}`

If so, yes it might be a good idea! If you mean any further above, I think it would be best to first keep the introductory sentence, and then have the prerequisites, but that's just my opinion

Reply:
agreed. just above the config_flow.

Reply:
Done